alacritty/src/logging.rs

```diff
@@ -27,6 +28,18 @@ pub const LOG_TARGET_IPC_CONFIG: &str = "alacritty_log_ipc_config";
 /// Name for the environment variable containing the log file's path.
 const ALACRITTY_LOG_ENV: &str = "ALACRITTY_LOG";
 
+/// Name for the environment varibale containing extra logging targets.
+///
+/// The targets are semicolon separated.
+const ALACRITTY_EXTRA_LOG_TARGETS_ENV: &str = "ALACRITTY_EXTRA_LOG_TARGETS";
+
+/// User configurable extra log targets to include.
+static EXTRA_LOG_TARGETS: Lazy<Option<Vec<&str>>> = Lazy::new(|| {
+    // The logging should be valid for the static lifetime.
+    let targets = Box::leak(env::var(ALACRITTY_EXTRA_LOG_TARGETS_ENV).ok()?.into_boxed_str());
+    Some(targets.split(';').collect())
+});
```

Comment:
Can we not use a String here?

Reply:
Do you mean using `String` as a whole and search in split string? I don't understand what do you mean exactly.

Reply:
I'm just a bit confused about the necessity of the boxed string leakage.

Reply:
You can't leak the string in rust directly yet, so you have to do something like that.

Reply:
You can use `Lazy<Option<Vec<String>>>` though, right?

Reply:
I can, but it'll mean to re-allocate everything without changing the semantics. It's just less efficient for the same thing.